From a2486611665b0f0985767ce264a751f1773c40fc Mon Sep 17 00:00:00 2001
From: Stefan Eichenberger <eichest@gmail.com>
Date: Mon, 17 Jul 2023 11:31:53 +0200
Subject: [PATCH] fbcon: show logo even if loglevel is <= quiet

Commit 10993504d647 ("fbcon: Silence fbcon logo on 'quiet' boots") will
make the kernel not show the logo when the command line parameter quiet
is used. This commit reverts it because this is what barebox does on the
medusa hardware.
---
 drivers/video/fbdev/core/fbcon.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/video/fbdev/core/fbcon.c b/drivers/video/fbdev/core/fbcon.c
index 24035b4f2cd7..93b463b79f9c 100644
--- a/drivers/video/fbdev/core/fbcon.c
+++ b/drivers/video/fbdev/core/fbcon.c
@@ -1004,9 +1004,6 @@ static void fbcon_init(struct vc_data *vc, int init)
 
 	info = fbcon_info_from_console(vc->vc_num);
 
-	if (logo_shown < 0 && console_loglevel <= CONSOLE_LOGLEVEL_QUIET)
-		logo_shown = FBCON_LOGO_DONTSHOW;
-
 	if (vc != svc || logo_shown == FBCON_LOGO_DONTSHOW ||
 	    (info->fix.type == FB_TYPE_TEXT))
 		logo = 0;
-- 
2.43.0

